tests/lib/containers/containers.py

```diff
@@ -6,6 +6,42 @@
 import pandas
 from dftimewolf.lib.containers import containers
 
+CONTAINER_CLASSES = [
```

Comment:
[Can save for a future PR] Could we dynamically generate this list somehow? So when new containers are created, the author doesn't need to know it should be added here.

Reply:
Opened https://github.com/log2timeline/dftimewolf/issues/674